workspaces/src/network/mod.rs

```diff
@@ -151,6 +154,18 @@ where
     ) -> ImportContractTransaction<'a, 'b> {
         ImportContractTransaction::new(id.to_owned(), worker.client(), self.client())
     }
+
+    async fn fast_forward(&self, delta_height: u64) -> anyhow::Result<()> {
+        // NOTE: RpcSandboxFastForwardResponse is an empty struct with no fields, so don't do anything with it:
+        let _forward_resp = self
```

Comment:
```suggestion
        self
```

If you're not checking the response serialization is correct, probably no need to bind this to a variable

Reply:
yeah, was binding to a variable to point out to readers that the resp object is empty, but it's not a big deal, so I'll just get rid of it